packages/semantic-release-config/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "private": true,
+  "name": "@hig/semantic-release-config",
+  "version": "0.1.0",
+  "description": "Semantic Release configuration for HIG packages",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@semantic-release/changelog": "^2.0.2",
+    "@semantic-release/git": "^5.0.0",
+    "@semantic-release/github": "^4.1.0",
```

Comment:
Does this plugin get used? The examples I'm looking at include this in a publish step in the semantic release config.